src/bindgen/ir/cfg.rs

```diff
@@ -167,6 +198,64 @@ impl Cfg {
         }
     }
 
+    fn matched_defines(&self, config: &Config, unmatched: &mut Vec<Cfg>) -> Option<Cfg> {
```

Comment:
This could get some comments :P

Reply:
Yes, some comments here would help. The method seems sound though.